src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -113,23 +116,14 @@ public String toString()
         private TargetOutputEstablishedStream()
         {
             this.streamState = this::beforeBegin;
-            this.throttleState = this::throttleSkipNextWindow;
         }
 
-        private void handleStream(
-            int msgTypeId,
-            MutableDirectBuffer buffer,
-            int index,
-            int length)
+        private void handleStream(int msgTypeId, MutableDirectBuffer buffer, int index, int length)
```

Comment:
Please revert to the original layout of method parameters.
We need to update the checkstyle to catch this.